README.md

```diff
@@ -38,8 +38,8 @@ Events are defined on the contract's code and can be emited by write methods at
 ```
 git clone https://github.com/LedgerHQ/cream-workshop.git
 ```
-2. Install dependencies with `yarn`
-3. Run the **CLI** a first time with no arguments to display your public address and Matic balance `yarn start`
+2. Install dependencies with `yarn` (or `npm`)
+3. Run the **CLI** a first time with no arguments to display your public address and Matic balance `yarn start` (or `npm run start`)
```

Comment:
I suppose that some people won't have yarn installed locally so I think it's important to mention that they can use npm.